microcosm_pubsub/consumer.py

```diff
@@ -100,12 +102,23 @@ def nack(self, message, visibility_timeout_seconds=None):
 
         """
         timeout = self.backoff_policy.compute_backoff_timeout(message, visibility_timeout_seconds)
+        if timeout < 0:
+            return self.skip_nack(message)
```

Comment:
Raising a `SkipMessage` at this point in the flow doesn't work, because it's not caught. Instead we have to `ack` the message.

Reply:
Can we change the behavior somehow so `SkipMessage` works?